Slim/Formats/XML.pm

```diff
@@ -428,15 +434,7 @@ sub parseRSS {
 			$url = $image->{'link'};
 		}
 
-		$feed{'image'} = $url unless ref $url; # scalar value only !
-	}
-	elsif ( ref $xml->{'itunes:image'} eq 'HASH' ) {
-		my $href = $xml->{'itunes:image'}->{'href'};
-		$feed{'image'} = $href unless ref $href;
-	}
```

Comment:
This branch effectively has been removed. On purpose?

Reply:
> Looking ok - except for that one image version that's missing now. I do understand this is to align with Apple's guidelines. But the motivation for this PR overall seems to be fixing issues caused by incorrect feeds. Shouldn't we leave that exception in there? It might have been added for this very reason: some providers set up their feeds incorrectly.

Fair point. I've just withdrawn this commit from the PR.